config/config.exs

```diff
@@ -8,7 +8,12 @@
 import Config
 
 config :better_words,
-  ecto_repos: [BetterWords.Repo]
+  ecto_repos: [BetterWords.Repo],
+  generators: [binary_id: true]
```

Comment:
This tells our generators that we will be using uuids so that when it creates a new resource, the database migration will set the primary key up correctly.

Reply:
Thanks for the details. To check my understanding, writing below:
Config module invokes config(root_key, opts) function. In this case:
- root_key as ":better_words"
- opts as [ecto_repos: [BetterWords.Repo], generators: [binary_id: true]]